spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigServicePropertySourceLocator.java

```diff
@@ -134,8 +134,8 @@ public ConfigServicePropertySourceLocator(ConfigClientProperties defaultProperti
 		}
 		if (properties.isFailFast()) {
 			throw new IllegalStateException(
-					"Could not locate PropertySource and the fail fast property is set, failing",
-					error);
+					"Could not locate PropertySource and the fail fast property is set, failing" +
+						(errorBody == null ? " " : ": " + errorBody), error);
```

Comment:
Why the space if there is no `errorBody`?

Reply:
@ryanjbaxter 
That's right, space was deleted.